src/python/espressomd/shapes.py

```diff
@@ -22,6 +22,28 @@ class Cylinder(ScriptInterfaceHelper):
     """
     _so_name = "Shapes::Cylinder"
 
+@script_interface_register
+class Ellipsoid(ScriptInterfaceHelper):
+    """
+    An ellipsoid.
+
+    For now only ellipsoids of revolution are supported.
+    The symmetry axis is aligned parallel to the x-direction.
+
+    Attributes
+    ----------
+    center : :obj:`array_like`
+       Coordinates of the center of the ellipsoid.
```

Comment:
Please horizontally align description and type of the attributes.

Reply:
The indentation in this file is a result of using autopep8, so I think it should not be changed manually.